Makefile

```diff
@@ -44,6 +44,9 @@ NETGO_CHECK=@strings $@ | grep cgo_stub\\\.go >/dev/null || { \
 	echo "    sudo go install -tags netgo std"; \
 	false; \
 }
+ENSURE_TOOLS=@if [ ! -d .tools ]; then \
+	git clone https://github.com/weaveworks/tools.git .tools/;\
+fi
```

Comment:
That seems like an odd way to do this. Why not just have a rule with `.tools` as a target and the `git clone` as the action?